knowledge-base/kb-modifying-exported-xlsx-gridview-spreadsheet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Modifying GridView Exported XLSX Spreadsheet
+description: How to make cusomt changes to exported spreadsheet document when exporting GridView to Excel xlsx.
+type: how-to
+page_title: Customize Excel XLSX Spreadhseet After Exporting From GridView
+slug: kb-modifying-gridview-exported-xlsx-spreadsheet
+position: 
+tags: WPF, GridView, RadGridView, Export, Exporting, Excel, Spreadsheet, xlsx, Customizing
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>2019.3.1023</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>RadGridView for WPF</td>
+	</tr>
+</table>
+
+
+## Description
+
+There are several ways to export the RadGridView as a spreadsheet. This article shows you how to leverage the RadGridView's `ExportToWorkbook` method to make changes to the spreadsheet before saving as Excel xlsx file.
+
+This article will show you how to add an extra row at the top of the speadsheet and insert a title
+
+## Solution
+
+The first step is to invoke the GridView's **ExportToWorkbook** method, optionally passing a `GridViewDocumentExportOptions` parameter.
+
+```csharp
+var exportOptions = new GridViewDocumentExportOptions()
+{
+    ShowColumnFooters = true,
+    ShowColumnHeaders = true,
+    ShowGroupFooters = true,
+    ShowGroupHeaderRowAggregates = true
+};
+
+Workbook workbook = MyGridView.ExportToWorkbook(exportOptions);
+```
+
+With the `Workbook` exported, you can now use the features of the Telerik Document Processing Libraries' **RadSpreadProcessing** to modify the document. 
+
+In this example, we are inserting a new row above the exported data to add a title.
+
+```csharp
+// Step 1. Get the first worksheet
+Worksheet worksheet = workbook.ActiveWorksheet;
+
+// Step 2. Insert a new row at the top
+worksheet.Rows.Insert(0);
+
+// Step 3. Select the worksheet's **A4** cell
+CellSelection selection = worksheet.Cells[0, 4];
+
+// Step 4. Insert the text "GridView Export" into the selected cell
+selection.SetValue("GridView Export");
+```
+
+Finally, we can now save it as an xlsx file using the `XlsxFormatProvider`.
+
+```csharp
+var dialog = new SaveFileDialog
+{
+    DefaultExt = "xlsx",
+    Filter = $"Excel files (.xlsx)|.xlsx|All files (.)|.",
+    FilterIndex = 1,
+    FileName = "GridView Export"
+};
+
+if (dialog.ShowDialog() == true)
+{
+    using (var output = dialog.OpenFile())
+    {
+        new XlsxFormatProvider().Export(workbook, output);
+    }
+}
+```
+
+## Resources
+
+For more settings and to gain an better understanding of what the  linker does, visit the following documentation article:
```

Comment:
remove space before "linker"

Reply:
The entire line was invalid and a copy/pasta fail on my part.